web/pdf_outline_view.js

```diff
@@ -69,6 +71,51 @@ var PDFOutlineView = (function PDFOutlineViewClosure() {
       };
     },
 
+    /**
+     * Prepend a button before an outline item which allows the user to toggle
+     * the visibility of all outline items at that level.
+     *
+     * @private
+     */
+    _addToggleButton: function PDFOutlineView_addToggleButton(div) {
+      var toggler = document.createElement('div');
+      toggler.className = 'outlineItemToggler';
+      toggler.onclick = function(event) {
```

Comment:
There are more examples, so I think this is fine; see e.g.
 - https://github.com/mozilla/pdf.js/blob/master/web/annotations_layer_builder.js#L53
 - https://github.com/mozilla/pdf.js/blob/master/web/annotations_layer_builder.js#L66
 - https://github.com/mozilla/pdf.js/blob/master/web/pdf_attachment_view.js#L65